### ISSUE TITLE
sig#779 - Upgrade app dependencies

### ISSUE THREAD

**Issue by evegufy** [eclipse-tractusx/portal-backend#1019]
As part of the regular [maintenance](https://github.com/eclipse-tractusx/sig-release/issues/779) the dependencies need to be upgraded.

TODO's:

- [x] upgrade dependencies - use existing dependabot PRs where possible, upgrade xunit* dependencies manually due to [dependabot limitation](https://github.com/eclipse-tractusx/portal-backend/blob/main/.github/dependabot.yml#L57) - and solve possible issues which araise from the upgrade
- [x] open IP review requests at the Eclipse Foundations GitLab for this dependencies which the Dash Tool signals accordingly
- [x] monitor IP review requests and answer questions if needed
- [x] update DEPENDENCIES file once all IP review requests have been approved

GH actions and docker are out of scope for this issue

xunit*:
xunit
xunit.runner.visualstudio
Xunit.Extensions.AssemblyFixture

**Comment by Phil91** [comment]
@evegufy all but xunit related packages have been updated.

**Comment by evegufy** [comment]
@Phil91 keeping the issue open for xunit upgrade, once IP issue is approved

**Comment by evegufy** [comment]
@Phil91 by now, it's too late to get the xunit upgrade into the release, please follow up on this as part of this issue https://github.com/eclipse-tractusx/portal-backend/issues/1148
